src/content/docs/zh-cn/guides/integrations-guide/preact.mdx

```diff
@@ -120,6 +120,41 @@ export default defineConfig({
 目前，`compat` 选项只适用于 React 库，将代码以 ESM 的形式导出。如果在构建时出现错误，请尝试将该库添加到你的`vite.ssr.noExternal: ['the-react-library']` 的 `astro.config.mjs` 文件中。
 :::
 
+## 选项
+
+### 组合对个 JSX 框架
+
+当你在同一个项目中使用多个 JSX 框架（React、Preact、Solid）时，Astro 需要确定每个组件应该使用哪个 JSX 框架特定的转换。如果你只向你的项目中添加了一个 JSX 框架集成，那么就不需要额外的配置。
```

Comment:
Original: `Astro needs to determine which JSX framework-specific transformations should be used for each of your components.`

No suggestion on top of my head yet, but the current translation seems too verbose and literal to me.